### ISSUE TITLE
After Scrutiny (full access) update from 0.7.3 to 0.8.0 scrutiny cannot start at all [BUG]

### ISSUE THREAD

**Issue by peanut-hu** [AnalogJ/scrutiny#606]
**Describe the bug**
After Scrutiny update from 0.7.3 to 0.8.0 scrutiny cannot start at all. 
Starting add-on fails.
Restarting complete HA also does not solve the problem. 
Removing and reinstalling newest version from scratch also does not work :(
After restoring partial backup of 0.7.3 version (and restarting HA), i get a "503: Service Unavailable", but after waiting some minutes, the 0.7.3 version starts to work again.

**Expected behavior**
I simply expected it to work like it did before :)

**Screenshots**
![image](https://github.com/AnalogJ/scrutiny/assets/120651312/bf015662-f2e5-446e-8710-18a85607257f)

**Log Files**
If related to missing devices or SMART data, please run the `collector` in DEBUG mode, and attach the log file.
See [/docs/TROUBLESHOOTING_DEVICE_COLLECTOR.md](docs/TROUBLESHOOTING_DEVICE_COLLECTOR.md) for other troubleshooting tips. 

Scrutiny log:
```
exec /init: too many levels of symbolic links
```
The log files will be available on your host in the `config` directory. Please attach them to this issue. 

`docker info`
```
Client:
 Version:    25.0.3
 Context:    default
 Debug Mode: false

Server:
 Containers: 21
  Running: 20
  Paused: 0
  Stopped: 1
 Images: 22
 Server Version: 24.0.7
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: journald
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version:
 runc version:
 init version:
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 6.6.20-haos
 Operating System: Home Assistant OS 12.1
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 3.625GiB
 Name: homeassistant
 ID: 7D2N:7PR5:MPVX:CFRJ:GW5E:EPI7:6YQQ:2VMN:WDLQ:HPZ7:HZSD:7GRQ
 Docker Root Dir: /mnt/data/docker
 Debug Mode: false
 Experimental: true
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

**Comment by martadinata666** [comment]
https://github.com/AnalogJ/scrutiny/issues/605

**Comment by sezlony** [comment]
it's now available so try upgrading to v0.8.0-2, that solves the notstarting issue 👍

**Comment by peanut-hu** [comment]
> it's now available so try upgrading to v0.8.0-2, that solves the notstarting issue 👍

Thank you, it works now! :)

**Comment by peanut-hu** [comment]
v0.8.0-2 solved the error.
